### ISSUE TITLE
Pytorch version:

### ISSUE THREAD

**Issue by xinyu1214** [amdegroot/ssd.pytorch#408]
Pytorch version: 

```
>>> import torch
>>> print(torch.__version__)
1.1.0
```

Python version:
```
Python 3.6.7 (default, Oct 22 2018, 11:32:17)
[GCC 8.2.0] on linux
```

multibox_loss.py: 
```
Switch the two lines 97,98:
loss_c = loss_c.view(num, -1)
loss_c[pos] = 0 # filter out pos boxes for now
```

```
Change line114 
N = num_pos.data.sum() -> N = num_pos.data.sum().double()
```

```
and change the following two lines to: 
loss_l = loss_l.double()
loss_c = loss_c.double()
```
train.py
```
loss_l.data[0] >> loss_l.data 
loss_c.data[0] >> loss_c.data 
loss.data[0] >> loss.data
```

And here is my output: 

```
timer: 11.9583 sec.
iter 0 || Loss: 11728.9388 || timer: 0.2955 sec.
iter 10 || Loss: nan || timer: 0.2843 sec.
iter 20 || Loss: nan || timer: 0.2890 sec.
iter 30 || Loss: nan || timer: 0.2934 sec.
iter 40 || Loss: nan || timer: 0.2865 sec.
iter 50 || Loss: nan || timer: 0.2855 sec.
iter 60 || Loss: nan || timer: 0.2889 sec.
iter 70 || Loss: nan || timer: 0.2857 sec.
iter 80 || Loss: nan || timer: 0.2843 sec.
iter 90 || Loss: nan || timer: 0.2835 sec.
iter 100 || Loss: nan || timer: 0.2846 sec.
iter 110 || Loss: nan || timer: 0.2946 sec.
iter 120 || Loss: nan || timer: 0.2860 sec.
iter 130 || Loss: nan || timer: 0.2846 sec.
iter 140 || Loss: nan || timer: 0.2962 sec.
iter 150 || Loss: nan || timer: 0.2989 sec.
iter 160 || Loss: nan || timer: 0.2857 sec.
```

_Originally posted by @kleinash in https://github.com/amdegroot/ssd.pytorch/issues/173#issuecomment-526295317_

**Comment by Wangxy1202** [comment]
hi~ which version cuda ?

**Comment by HongChow** [comment]
> Pytorch version:
> 
> ```
> >>> import torch
> >>> print(torch.__version__)
> 1.1.0
> ```
> 
> Python version:
> 
> ```
> Python 3.6.7 (default, Oct 22 2018, 11:32:17)
> [GCC 8.2.0] on linux
> ```
> 
> multibox_loss.py:
> 
> ```
> Switch the two lines 97,98:
> loss_c = loss_c.view(num, -1)
> loss_c[pos] = 0 # filter out pos boxes for now
> ```
> 
> ```
> Change line114 
> N = num_pos.data.sum() -> N = num_pos.data.sum().double()
> ```
> 
> ```
> and change the following two lines to: 
> loss_l = loss_l.double()
> loss_c = loss_c.double()
> ```
> 
> train.py
> 
> ```
> loss_l.data[0] >> loss_l.data 
> loss_c.data[0] >> loss_c.data 
> loss.data[0] >> loss.data
> ```
> 
> And here is my output:
> 
> ```
> timer: 11.9583 sec.
> iter 0 || Loss: 11728.9388 || timer: 0.2955 sec.
> iter 10 || Loss: nan || timer: 0.2843 sec.
> iter 20 || Loss: nan || timer: 0.2890 sec.
> iter 30 || Loss: nan || timer: 0.2934 sec.
> iter 40 || Loss: nan || timer: 0.2865 sec.
> iter 50 || Loss: nan || timer: 0.2855 sec.
> iter 60 || Loss: nan || timer: 0.2889 sec.
> iter 70 || Loss: nan || timer: 0.2857 sec.
> iter 80 || Loss: nan || timer: 0.2843 sec.
> iter 90 || Loss: nan || timer: 0.2835 sec.
> iter 100 || Loss: nan || timer: 0.2846 sec.
> iter 110 || Loss: nan || timer: 0.2946 sec.
> iter 120 || Loss: nan || timer: 0.2860 sec.
> iter 130 || Loss: nan || timer: 0.2846 sec.
> iter 140 || Loss: nan || timer: 0.2962 sec.
> iter 150 || Loss: nan || timer: 0.2989 sec.
> iter 160 || Loss: nan || timer: 0.2857 sec.
> ```
> 
> _Originally posted by @kleinash in [#173 (comment)](https://github.com/amdegroot/ssd.pytorch/issues/173#issuecomment-526295317)_

fixed?

**Comment by up2m** [comment]
I have the same problem，do you solve it?

**Comment by jmu201521121021** [comment]
@Summar-skyI solved it by reducing the learning rate.

**Comment by chnzhero** [comment]
> @Summar-skyI solved it by reducing the learning rate.

it works, thx

**Comment by elcolie** [comment]
@jmu201521121021 Thank you
`--lr 1e-5` works.
